src/toplevel.pl

```diff
@@ -1,5 +1,4 @@
-:- module('$toplevel', [argv/1,
-                        copy_term/3]).
+:- module('$toplevel', [copy_term/3, started/0]).
```

Comment:
why is `started/0` public?

Reply:
`loader.pl` uses it and I thought of leaving it as explicit as before

Reply:
I think `loader.pl` uses the module-qualified version, so there seems no need to export such a "normal" (i.e., prone to naming conflicts, even though no program likely needs it) name?